### ISSUE TITLE
Uncaught TypeError: Cannot read property 'name' of undefined

### ISSUE THREAD

**Issue by ArtemyB** [ionide/ionide-fsharp#174]
This error appears when I try to open the markdown preview for markdown-file ("markdown-preview" Atom package). And it also blocks the tab with markdown preview so I can't choose it (it just appears, but it doesn't open after clicking on it). The error disappears and markdown preview works correctly if I uninstall ionide-fsharp package.

**Atom Version**: 1.0.19
**System**: Windows 8.1 Single Language
**Thrown From**: [ionide-fsharp](https://github.com/ionide/ionide-fsharp) package, v1.4.2

### Stack Trace

Uncaught TypeError: Cannot read property 'name' of undefined

```
At C:\Users\Dark_Clark\AppData\Local\atom\app-1.0.19\resources\app.asar\node_modules\q\q.js:128

TypeError: Cannot read property 'name' of undefined
  at ViewsHelpers__isFSharpEditor$ (C:\Users\Dark_Clark\.atom\packages\ionide-fsharp\lib\fsharp.js:5412:102)
  at TooltipHandler__initialize$ (C:\Users\Dark_Clark\.atom\packages\ionide-fsharp\lib\fsharp.js:5050:9)
  at C:\Users\Dark_Clark\.atom\packages\ionide-fsharp\lib\fsharp.js:4963:14
  at Emitter.module.exports.Emitter.emit (C:\Users\Dark_Clark\AppData\Local\atom\app-1.0.19\resources\app.asar\node_modules\event-kit\lib\emitter.js:82:11)
  at C:\Users\Dark_Clark\AppData\Local\atom\app-1.0.19\resources\app.asar\src\pane-container.js:320:34
  at Emitter.module.exports.Emitter.emit (C:\Users\Dark_Clark\AppData\Local\atom\app-1.0.19\resources\app.asar\node_modules\event-kit\lib\emitter.js:82:11)
  at Pane.module.exports.Pane.setActiveItem (C:\Users\Dark_Clark\AppData\Local\atom\app-1.0.19\resources\app.asar\src\pane.js:250:22)
  at Pane.module.exports.Pane.activateItem (C:\Users\Dark_Clark\AppData\Local\atom\app-1.0.19\resources\app.asar\src\pane.js:314:21)
  at C:\Users\Dark_Clark\AppData\Local\atom\app-1.0.19\resources\app.asar\src\workspace.js:489:16
  at _fulfilled (C:\Users\Dark_Clark\AppData\Local\atom\app-1.0.19\resources\app.asar\node_modules\q\q.js:794:54)
  at self.promiseDispatch.done (C:\Users\Dark_Clark\AppData\Local\atom\app-1.0.19\resources\app.asar\node_modules\q\q.js:823:30)
  at Promise.promise.promiseDispatch (C:\Users\Dark_Clark\AppData\Local\atom\app-1.0.19\resources\app.asar\node_modules\q\q.js:756:13)
  at C:\Users\Dark_Clark\AppData\Local\atom\app-1.0.19\resources\app.asar\node_modules\q\q.js:817:14
  at flush (C:\Users\Dark_Clark\AppData\Local\atom\app-1.0.19\resources\app.asar\node_modules\q\q.js:110:17)
  at doNTCallback0 (node.js:416:9)
  at process._tickCallback (node.js:345:13)

```

### Commands

```
     -3:24 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:05.6.0 markdown-preview:toggle (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "themes": [
      "atom-material-ui",
      "atom-material-syntax"
    ],
    "destroyEmptyPanes": false,
    "disabledPackages": [
      "atom-beautify",
      "haskell-grammar",
      "autocomplete-python"
    ]
  },
  "ionide-fsharp": {}
}
```

### Installed Packages

```coffee
# User
atom-material-syntax, v0.2.4
atom-material-ui, v0.6.2
atom-yeoman, v0.3.15
autocomplete-haskell, v0.6.0
autocomplete-plus-python-jedi, v0.3.6
browser-plus, v0.0.45
character-table, v0.4.3
color-picker, v2.0.13
column-select, v0.2.0
file-icons, v1.6.11
haskell-ghc-mod, v1.0.0
haskell-pointfree, v0.1.2
hover-tooltips-hdevtools, v0.8.0
ide-haskell-cabal, v1.0.0
ionide-fake, v1.2.0
ionide-fsharp, v1.4.2
ionide-fsi, v1.0.8
ionide-installer, v1.3.0
ionide-paket, v2.2.0
ionide-webview, v1.0.0
ionide-yeoman, v1.0.1
language-haskell, v1.4.11
language-pgsql, v0.2.0
linter, v1.9.1
linter-hdevtools, v1.6.0
linter-hlint, v0.4.2
minimap, v4.15.0
Sublime-Style-Column-Selection, v1.3.0
tabs-to-spaces, v0.11.1
webbox-color, v0.5.9

# Dev
No dev packages
```

**Comment by 7sharp9** [comment]
I just got this while editing a rust file.

**Comment by Krzysztof-Cieslak** [comment]
Should be fixed in `1.4.4`
